test/e2e/installplan_e2e_test.go

```diff
@@ -2715,6 +2721,474 @@ var _ = Describe("Install Plan", func() {
 			Expect(ref.Kind).To(Equal("ConfigMap"))
 		}
 	})
+
+	It("limits installed resources if the scoped serviceaccount has no permissions", func() {
+		c := newKubeClient()
+		crc := newCRClient()
+
+		By("creating a scoped serviceaccount specifified in the operatorgroup")
```

Comment:
Nit: Typo, but it's coming from #1888 so we can ignore it here.
```suggestion
		By("creating a scoped serviceaccount specified in the operatorgroup")
```

Reply:
Good catch -- will see if I can fix in the 4.5 backport